core/controllers/admin.py

```diff
@@ -379,6 +379,8 @@ def _load_dummy_new_structures_data(self):
             question_id_1 = question_services.get_new_question_id()
             question_id_2 = question_services.get_new_question_id()
             question_id_3 = question_services.get_new_question_id()
+            question_id_4 = question_services.get_new_question_id()
```

Comment:
Just curious -- what's the rationale for adding stuff here, and why are you (below) making them all have the same skill ID? The second change seems to reduce the utility of the generated data a bit, and it conflicts with the docstring. So I kind of suggest leaving things as they are -- unless you have a good reason to change them?

Reply:
I had added these to test the practice tab (it shows only if there are 5 questions). But I don't think this is required to be done in this PR. I will revert these changes.

Reply:
Done.